src/KubernetesClient/Kubernetes.ConfigInit.cs

```diff
@@ -117,9 +117,9 @@ private void CreateHttpClient(DelegatingHandler[] handlers, KubernetesClientConf
 #if NET5_0_OR_GREATER
             FirstMessageHandler = HttpClientHandler = new SocketsHttpHandler
```

Comment:
SocketsHttpHandler has so many props
why not create a `callback/factory` and you can put socket callback there as well

Reply:
Done